user/enterprise/ssl-certificate-management.md

```diff
@@ -0,0 +1,268 @@
+
+> In all cases of using self-signed certificates you must [disable SSL certificate validation](#disabling-ssl-verification) in your settings. This can be done after installation is complete.
+
+> If you decide at a later time to upload your own custom certificate then this can be done at any time via the [admin dashboard](#how-to-add-or-update-certificates-after-installation).
+
+#### Trial Users
+
+If you are using a trial license to evaluate Travis CI Enterprise then going through the effort of obtaining a trusted certificate might not make sense at this time. For this scenario we offer the ability to use self-signed certificates that are generated by Travis CI Enterprise itself.
+
+To do this you can select the "Use Self-Signed Cert" option when prompted to provide a custom SSL certificate during installation.
+
+#### Organizations with an internally managed certificate authority
+
+Some organizations manage their own internal certificate authority. An internal authority may or may use a root certificate provided by a [trusted root authority](https://en.wikipedia.org/wiki/Certificate_authority).
+
+#### Without a root certificate from a trusted CA
+
+If your internal CA is not using a root certificate provided by a trusted CA then any certificates produced are considered to be 'untrusted' by most tools and browsers, same as a self-signed certificate.
+
+Certificates created by an internal CA can be uploaded during installation in the same way as any other certificate.
+
+Please be aware that you will also need to be sure to follow the [instructions on disabling SSL verification](#disabling-ssl-verification) below in order to use Travis CI Enterprise.
+
+#### With a root certificate from a trusted CA
+
+If your organization's internal CA does use a root certificate purchased from a trusted CA then the above will not apply. You will not need to disable SSL verification and can proceed as if you are using a certificate directly from a trusted CA.
+
+> In some cases it may be necessary to include the intermediate certificate chain in the settings in order for SSL verification to work correctly. The reasons for this can vary depending on your internal setup and your root certificate provider. Please see the section below on [how to set intermediate certificate chains](#using-intermediate-certificate-chains) for instructions.
+
+### Disabling SSL Verification
+
+> SSL verification helps ensure that all traffic between Travis CI Enterprise components is protected. Disabling SSL verification has serious implications!
+
+You can disable SSL verification across Travis CI Enterprise via the following steps:
+
+- Log into your dashboard at `https://<your-travis-ci-enterprise-domain>:8800`.
```

Comment:
Use a numbered list here, for steps in order.

Reply:
applies throughout

Reply:
Addressed in 51e00208.